src/pages/de/core-concepts/framework-components.md

```diff
@@ -0,0 +1,234 @@
+
+## Interaktive Komponenten hydratisieren
+
+Eine Framework-Komponenten kann kann aktiviert (hydratisiert) werden, indem die `client:*`-Direktiven verwendet werden. Das ist ein Komponenten-Attribut, welches definiert wie die Komponente **gerendert** und **hydriert** werden soll.
+
+Die [Client-Direktive](/en/reference/directives-reference/#client-directives) beschreibt ob die Komponente zur Build-Zeit gerendet werden soll und wann das JavaScript der Komponente client-seitig durch den Browser geladen werden soll. 
+
+Die meisten Direktiven rendern die Komponenten auf dem Server zur Build-Zeit. Das Komponenten-JS wird entsprechend der angewendeten Direktive an den Client gesendet. Die Komponente wird hydratisiert, wenn ihr JS geladen wurde.
+
+```astro title="src/pages/interactive-components.astro" /client:\S+/
+---
+// Beispiele: Framework-Komponenten im Browser hydratisiert.
+import InteractiveButton from '../components/InteractiveButton.jsx';
+import InteractiveCounter from '../components/InteractiveCounter.jsx';
+---
+<!-- Das JS dieser Komponente wird geladen, wenn die Seite lädt-->
+<InteractiveButton client:load />
+
+<!-- Das JS dieser Komponente wird nicht an den Client übertragen solange der User 
+ nicht auf der Seite herunterscrollt und die Komponente auf der Seite sichtbar ist -->
+<InteractiveCounter client:visible />
+```
+
+:::Achtung
+Jegliches JS, welches für das Framework der Komponente (z.B. React, Svelt) notwendig ist, wird mit der Seite geladen. Die `client:*`-Direktiven geben lediglich vor wann das _Komponenten JS_ geladen wird und wann die _Komponente_ hydratisiert wird.
+:::
+
+:::Hinweis[Barrierefreiheit]
+Die meisten Framework-spezifischen Barrierefreiheit-Pattern sollten genauso funktionieren, wenn Sie in Astro verwendet werden. Vergewissere dich, dass du eine client-Direktive verwendest, die sicherstellt, dass das JavaScript für die Barrierefreiheit korrekt geladen und zur richtigen Zeit ausgeführt wird!
+:::
+
+### Verfügbare Hydratisierungs-Direktiven 
+
+Es sind einige Hydratisierungs-Direktiven für UI Framework-Komponenten verfügbar: `client:load`, `client:idle`, `client:visible`, `client:media={QUERY}` and `client:only={FRAMEWORK}`.
+
+📚 Schaue unsere [Direktiven-Referenz](/en/reference/directives-reference/#client-directives) Seite für die vollständige Beschreibung der Direktiven sowie deren Nutzung an.
+
+## Frameworks mischen
+
+Du kannst Komponenten aus verschiedenen Frameworks in der selben Astro-Komponente importieren und rendern.
+
+```astro title="src/pages/mixing-frameworks.astro"
+---
+// Beispiel: Mehrere Framework-Komponenten auf der gleichen Seite verwenden.
+import MyReactComponent from '../components/MyReactComponent.jsx';
+import MySvelteComponent from '../components/MySvelteComponent.svelte';
+import MyVueComponent from '../components/MyVueComponent.vue';
+---
+<div>
+  <MySvelteComponent />
+  <MyReactComponent />
+  <MyVueComponent />
+</div>
+```
+
+:::Achtung
```

Comment:
```suggestion
:::caution
```

This will automatically be displayed as "Achtung" because there are default translations for our aside captions. As explained above, translating the directive name does not work.